app/lib/features/chat/pages/room_profile_page.dart

```diff
@@ -327,18 +326,18 @@ class _RoomProfilePageState extends ConsumerState<RoomProfilePage> {
                 padding:
                     const EdgeInsets.symmetric(horizontal: 16, vertical: 8),
                 child: Text(
-                  '${L10n.of(context).members} (${list.length})',
+                  L10n.of(context).membersCountRight(list.length),
```

Comment:
Same as above. let's stick with one...
```suggestion
                  L10n.of(context).membersCount(list.length),
```